tests/aws/services/events/test_events_targets.snapshot.json

```diff
@@ -669,5 +669,104 @@
cute-api.<region>.amazonaws.com\"], \"User-Agent\": [\"aws-sdk-java/1.12.769 Linux/5.10.224-190.876.amzn2int.x86_64 OpenJDK_64-Bit_Server_VM/17.0.12+8-LTS java/17.0.12 scala/2.12.17 kotlin/1.5.32 vendor/Amazon.com_Inc. cfg/retry-mode/legacy\"], \"Via\": [\"1.1 a929b4bfaa0111e3feb7c4dbffdbd8d8.cloudfront.net (CloudFront)\"], \"X-Amz-Cf-Id\": [\"BKQQYBXKENj_W1ZK2LDm_YMuNXI1XHU9isWbrESHAhr5lfFoNcg0vg==\"], \"X-Amz-Date\": [\"20240919T193144Z\"], \"X-Amz-Security-Token\": [\"IQoJb3JpZ2luX2VjECwaCXVzLWVhc3QtMSJIMEYCIQDvt7mWI5G/D00yl23M9LKlTTPdeqQg49Vt7NJ8CxvBRQIhAIM/C7B3vq8swY/BwtVMQ1pRJGTkVLRlc2K91GP73/4OKuMCCGUQABoMNTM1MDAyODY2NjgyIgxSjjvKlfZx7lJImTwqwAJuXUDPlL7a9zlcLfiIa5IGDlKRzQKumT+0Hs0bcH4k90qBGqocF4CE785VDt70RSyTsdNY33m3v03W2e+D3gk+Fygkld5xasYKt6Uhd0FuoVFvNeY/dKhfQUVukvdCuc3aVo86KUHWQY2Ot2ZHKjXKsbnuiS57kVsgayS4sOMLuVsCv9hrSG5Ek85l/+TAdtOHekNU2hV5Bus2iPhsI7HL+vjgMCsdvZDz9Pq4tF56lnXEF41LJahHJc01nAd2RaNJysgWmEjKeylo0zTvKb17xWywGsb31cHonmnLj1bhB0nfzOBhwKZdzfEOVw54OR5BLLb1+14t4RmivjZfXoRlJUMKQ+mgVt1Td4k5aTUXN4vo9gcvN8GRQA7xuLHoGCZpTuEz0UyHSrG5596xtZ1pvilNAJnTBrkuRImmB2h3JTCg+LG3Bjq+AUOgEmpmLhONNqo12HBQg93wD87quqtbU5xaAxEKGngF2nG+oDV/iRYpkgP3jhCzYY0PyXdmPgYMVO6RJpB/mt5LyURq0qz9or/J0qbXW2aYufyqGnfxfx0Swcdr5dIPKWsu+YpScElR9LP33wiUZpvNWee/qcXO2S7Y8vHVTsOuRfml5W7Tlz53xfVgvVg9JA/Iv0LS2sYaiwKoGH+TC4tbXvBBBOVPcGFoUFu2wBC1XtVX+MAKRcxM0LTkfEM=\"], \"X-Amz-Source-Account\": [\"111111111111\"], \"X-Amz-Source-Arn\": [\"arn:<partition>:events:<region>:111111111111:rule/<resource:3>/<resource:4>\"], \"X-Amzn-Trace-Id\": [\"Root=1-66ec7c21-44b7c5793da6158925dd8353\"], \"X-Forwarded-For\": [\"72.21.217.42, 70.132.32.74\"], \"X-Forwarded-Port\": [\"443\"], \"X-Forwarded-Proto\": [\"https\"]}, \"queryStringParameters\": null, \"multiValueQueryStringParameters\": null, \"pathParameters\": null, \"stageVariables\": null, \"requestContext\": {\"resourceId\": \"7a07rd\", \"resourcePath\": \"/test\", \"httpMethod\": \"POST\", \"extendedRequestId\": \"eXhVNFPooAMELnA=\", \"requestTime\": \"19/Sep/2024:19:31:45 +0000\", \"path\": \"/test/test/\", \"accountId\": \"111111111111\", \"protocol\": \"HTTP/1.1\", \"stage\": \"test\", \"domainPrefix\": \"sp260dl21k\", \"requestTimeEpoch\": 1726774305007, \"requestId\": \"3ae58bde-9ef6-4867-af27-361dd207f942\", \"identity\": {\"cognitoIdentityPoolId\": null, \"accountId\": null, \"cognitoIdentityId\": null, \"caller\": null, \"sourceIp\": \"72.21.217.42\", \"principalOrgId\": null, \"accessKey\": null, \"cognitoAuthenticationType\": null, \"cognitoAuthenticationProvider\": null, \"userArn\": null, \"userAgent\": \"aws-sdk-java/1.12.769 Linux/5.10.224-190.876.amzn2int.x86_64 OpenJDK_64-Bit_Server_VM/17.0.12+8-LTS java/17.0.12 scala/2.12.17 kotlin/1.5.32 vendor/Amazon.com_Inc. cfg/retry-mode/legacy\", \"user\": null}, \"domainName\": \"sp260dl21k.execute-api.<region>.amazonaws.com\", \"deploymentId\": \"<id:1>\", \"apiId\": \"sp260dl21k\"}, \"body\": \"{\\\"message\\\": \\\"Hello from EventBridge\\\"}\", \"isBase64Encoded\": false}\n"
```

Comment:
Looks like the JSON is incorrectly formatted on snapshot because of that `"Received event:"` string ☹️